src/Sql/Sql/help/Get-AzSqlDeletedInstanceDatabaseBackup.md

```diff
@@ -0,0 +1,177 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/az.sql/get-azsqldeletedinstancedatabasebackup
+schema: 2.0.0
+---
+
+# Get-AzSqlDeletedInstanceDatabaseBackup
+
+## SYNOPSIS
+Gets a deleted database that you can restore.
+
+## SYNTAX
+
+```
+Get-AzSqlDeletedInstanceDatabaseBackup [-ResourceGroupName] <String> [-InstanceName] <String>
+ [[-DatabaseName] <String>] [[-DeletionDate] <DateTime>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzSqlDeletedInstanceDatabaseBackup** cmdlet gets a specified deleted SQL Instance database backup that you can restore, or all deleted backups that you can restore.
+This cmdlet is also supported by the SQL Instance Stretch Database service on Azure.
+
+## EXAMPLES
+
+### Example 1: Get all deleted database backups on a server
+```powershell
+PS C:\>Get-AzSqlDeletedInstanceDatabaseBackup -ResourceGroupName "ContosoResourceGroup" -InstanceName "ContosoServer"
+DeletionDate         : 2019-03-03 12:00:17 AM
+ResourceGroupName    : ContosoResourceGroup
+ManagedInstanceName  : ContosoServer
+Name                 : DB1
+CreationDate         : 2019-03-02 11:00:51 PM
+EarliestRestorePoint : 2019-03-02 11:05:30 PM
+Id                   : /subscriptions/ee5ea899-0791-418f-9270-77cd8273794b/resourceGroups/ContosoResourceGroup/providers/Microsoft.
```

Comment:
Make sure all information specific to your accounts is removed (ex: subscriptionid should be 00000000-0000-0000-0000-000000000000)